homeassistant/components/coolmaster/climate.py

```diff
@@ -66,15 +67,10 @@ def __init__(self, coordinator, unit_id, unit, supported_modes, info):
         self._hvac_modes = supported_modes
         self._info = info
 
-    def _refresh_from_coordinator(self):
+    @callback
+    def _handle_coordinator_update(self):
         self._unit = self.coordinator.data[self._unit_id]
-        self.async_write_ha_state()
-
-    async def async_added_to_hass(self):
-        """When entity is added to hass."""
-        self.async_on_remove(
-            self.coordinator.async_add_listener(self._refresh_from_coordinator)
-        )
+        super()._handle_coordinator_update()
```

Comment:
Why do we call super here?

Reply:
To execute the original logic of the CoordinatorEntity. It's currently just a call to `self.async_write_ha_state()` but maybe something else is added in the future.